rest-service/manager_rest/rest/resources_v3_1/secrets.py

```diff
@@ -213,6 +205,21 @@ def _import_secrets(self, secrets_list, tenant_map, passphrase,
                 self._import_secret(secret, colliding_secrets, override)
         return all_secrets_errors
 
+    def _check_timestamp_and_owner(self, secret, secret_errors):
+        if 'created_at' in secret or 'updated_at' in secret:
+            if not is_user_action_allowed('set_timestamp', None, True):
+                secret_errors['timestamp'] = \
+                    'You require the set_timestamp permission.'
```

Comment:
A bit awkward message. Maybe 'your user requires...'?

Reply:
That sounds more awkward I think?

Reply:
Modified after further out-of-band-discussion.